public/assets/librejs/librejs.html

```diff
@@ -1,84 +0,0 @@
-<!DOCTYPE html>
```

Comment:
[As requested](https://github.com/go-gitea/gitea/pull/2241#issuecomment-320909454) => the [VERSIONS](https://github.com/go-gitea/gitea/pull/2241/commits/926cb40203569545940cde0deb20aefc6897a34e) file